maps/randomvaults/objects.dm

```diff
@@ -174,7 +177,7 @@
 
 /obj/machinery/power/apc/no_alerts/vault_listening/initialize()
 	. = ..()
-	name = "\improper Listening Outpost APC."
+	name = "\improper Listening Outpost APC"
```

Comment:
It was sort of relevant because I copied that for a new APC and noticed the period in the name when I was testing